src/Sylius/Bundle/PaymentBundle/Doctrine/DBAL/Type/ObjectType.php

```
@@ -0,0 +1,63 @@
+
+    public function convertToPHPValue($value, AbstractPlatform $platform): mixed
+    {
+        if ($value === null) {
+            return null;
+        }
+
+        if (is_resource($value)) {
+            $value = stream_get_contents($value);
+        }
+
+        set_error_handler(static function (int $code, string $message): never {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$code'. (undefined)

(UnusedFormalParameter)